_data/simple-icons.json

```
@@ -10742,6 +10742,11 @@
             "source": "https://www.okta.com/press-room/media-assets/",
             "guidelines": "https://www.okta.com/terms-of-use-for-okta-content/"
         },
+        {
+            "title": "OKX",
+            "hex": "121212",
```

```suggestion
            "hex": "000000",
```

I'm seeing `#000000` as the primary background of the website and as the background color of the favicon and other promotional materials.

---

Fixed